### PR TITLE
Add zap to qq.app v4.0.2

### PR DESCRIPTION
**Pull request by githubutilities**

### CODE REVIEW

**Comment by githubutilities** [comment]
Also add zap to qqbrowser v2.3.2160

**Comment by vitorgalvao** [comment]
In the future, please submit separate pull requests for each cask, as more often than not this type of pull request brings problems with one or more of them, and it halts the inclusion of the other ones.

I’ll point out the mistakes in the casks.

File: Casks/qqbrowser.rb [hunk 1]
```diff
@@ -5,7 +5,20 @@
   url "http://dldir1.qq.com/invc/tt/QQBrowser_#{version.sub(%r{^.*\.},'')}.dmg"
   name 'QQ浏览器'
   homepage 'http://browser.qq.com/mac/'
-  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
+  license :commercial
 
   app 'QQBrowser.app'
+
+  uninstall :quit => 'com.tencent.QQBrowser'
+  zap :delete => [
```
Comment (vitorgalvao) [inline Casks/qqbrowser.rb]: There should be an empty line before this one.

File: Casks/qqbrowser.rb [hunk 2]
```diff
@@ -5,7 +5,20 @@
...
+                  "~/Library/Application Support/QQBrowser2",
+                  "~/Library/Caches/com.tencent.QQBrowser",
+                  "~/Library/Caches/com.tencent.QQBrowser.helper",
+                  "~/Library/Caches/QQBrowser2",
+                  "~/Library/Preferences/com.tencent.QQBrowser.helper.EH.plist",
+                  "~/Library/Preferences/com.tencent.QQBrowser.helper.plist",
+                  "~/Library/Preferences/com.tencent.QQBrowser.plist",
+                  "~/Library/Saved Application State/com.tencent.QQBrowser.savedState"
+                 ]
+
```
Comment (vitorgalvao) [inline Casks/qqbrowser.rb]: This empty line should go.

File: Casks/qqbrowser.rb [hunk 3]
```diff
@@ -5,7 +5,20 @@
...
   name 'QQ浏览器'
   homepage 'http://browser.qq.com/mac/'
-  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
+  license :commercial
 
   app 'QQBrowser.app'
+
+  uninstall :quit => 'com.tencent.QQBrowser'
+  zap :delete => [
+                  "~/Library/Application Support/QQBrowser2",
```
Comment (vitorgalvao) [inline Casks/qqbrowser.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these (and all subsequent ones) to single quotes.

**Comment by githubutilities** [comment]
OK. Thanks

**Pull request closed without merging**